metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -789,6 +860,7 @@ def gen_table_dataset_workunits(
         yield from self.gen_dataset_workunits(
             table=table,
             columns=columns,
+            table_constraints=table_constraints,
```

Comment:
Should we be adding table constraints to the table type instead of passing it explicitly everywhere?